src/stages/agent.ts

```diff
@@ -57,47 +62,313 @@ export class Agent implements Stage<Agent.Input, Agent.Output> {
 
   constructor() {
     this.lmBridge = new LmBridge(TEMPERATURE, false, TOOLS);
-    this.lmBridge; // TODO: remove this
   }
 
   async execute(
     input: Agent.Input,
     context: StageContext
   ): Promise<Agent.Output> {
-    const systemPrompt = await context.getPrompt("v2-agent");
-    const platformInfoPrompt = await context.getPrompt(
-      "v2-agent-platform-info",
-      {
+    const actions = await this.callLm(
+      input,
+      context,
+      await this.buildPrompt(context, input)
+    );
+
+    return {
+      actions,
+    };
+  }
+
+  private async buildPrompt(ctx: StageContext, input: Agent.Input) {
```

Comment:
```suggestion
  private async buildPrompt(ctx: StageContext, input: Agent.Input): Promise<PromptSet> {
```

PromptSet은 이미 있는 타입과 이름이 겹치니 다른 이름을 쓰면 좋겠습니다.

Reply:
internal로 써서 그냥 지어놓았는데
생각해보니 맥락 파악에 해로움을 주겠군요